### PR TITLE
[kesch] RH/6.8 PE/17.12

### PR DESCRIPTION
**Pull request by jgphpc**
Updated build list is here 
https://github.com/eth-cscs/production/blob/RH6.8-17.12-jg/jenkins-builds/MCH-RH7.3-17.02

### CODE REVIEW

**Comment by gppezzi** [comment]
@jgphpc I've updated the missing .eb files for gmvolf 17.02, only score-p and scalasca are missing ( and I'm still finalizing R).

**Comment by gppezzi** [comment]
R should be also ready, here's the current list (ready to be checked with MCH cc @amangili):
```
Autotools/201602
Bison/3.0.4
Bison/3.0.4-gmvolf-17.02
Boost/1.63.0-gmvolf-17.02-Python-2.7.13
CDO/1.7.2-gmvolf-17.02
CMake/3.7.2
FFTW/3.3.6-pl1-gmvapich2-17.02
GCC/5.4.0-binutils-2.27-libiberty
GDAL/2.1.3-gmvolf-17.02
GEOS/3.6.0-gmvolf-17.02
GMP/6.1.1-gmvolf-17.02
HDF/4.2.8-gmvolf-17.02
HDF5/1.8.18-gmvolf-17.02
Java/1.8.0_121
MVAPICH2/2.2-GCC-5.4.0-binutils-2.27-libiberty
NCO/4.6.2-gmvolf-17.02
OpenBLAS/0.2.19-GCC-5.4.0-binutils-2.27-libiberty-LAPACK-3.6.1
Python/2.7.13-gmvolf-17.02
Python/3.6.0-gmvolf-17.02
R/2.15.3-gmvolf-17.02-libX11-1.6.3
R/3.0.2-gmvolf-17.02-libX11-1.6.3
R/3.2.4-gmvolf-17.02-libX11-1.6.3
R/3.3.2-gmvolf-17.02
Ruby/2.4.0-gmvolf-17.02
ScaLAPACK/2.0.2-gmvapich2-17.02-OpenBLAS-0.2.19-LAPACK-3.6.1
Subversion/1.9.5
gmvapich2/17.02
gmvolf/17.02
ncview/2.1.7-gmvolf-17.02
netCDF/4.4.1.1-gmvolf-17.02
netCDF-C++/4.3.0-gmvolf-17.02
netCDF-Fortran/4.4.4-gmvolf-17.02
netcdf-python/1.2.7-gmvolf-17.02-Python-2.7.13
netcdf-python/1.2.7-gmvolf-17.02-Python-3.6.0
rstudio/0.98.1103
```

File: easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.2.19-GCC-5.4.0-binutils-2.27-libiberty-LAPACK-3.6.1.eb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    "http://www.netlib.org/lapack/",
+    "http://www.netlib.org/lapack/timing/",
+    "https://github.com/xianyi/OpenBLAS/archive/",
+]
+
+patches = [
+    (lapack_src, '.'),  # copy LAPACK tarball to unpacked OpenBLAS dir
+    (large_src, '.'),
+    (timing_src, '.'),
+#     'OpenBLAS-0.2.19-GCC-5.4.0-binutils-2.27-libiberty-LAPACK-3.5.0.patch',
```
Comment (gppezzi) [inline easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.2.19-GCC-5.4.0-binutils-2.27-libiberty-LAPACK-3.6.1.eb]: @jgphpc  can you explain why this is commented out?

**Comment by gppezzi** [comment]
retest this please

**Comment by gppezzi** [comment]
retest this please

**Comment by lucamar** [comment]
retest this please

**Pull request merged**